test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9455,6 +9457,30 @@ def forward(self, x):
         x = torch.randn(10, 5)
         self.run_test(M(), (x,))
 
+    def test_custom_opsets_triu(self):
```

Comment:
We already have tests for this argument in test_utility_funs.py.
Is the purpose of the new tests to verify ORT custom kernels? Or you are just testing the custom export functionality?